atf_core/src/atf_core/testblock.py

```diff
@@ -12,7 +12,6 @@ def __init__(self, name, metric_handles, recorder_handle):
         self.trigger = None
         self.timestamp = None
         self.exception = None
-        self.atf_started = False
         self.status = None
```

Comment:
this should be using ths SM status enum, i.e. `INACTIVE`, right?

Reply:
fixed in https://github.com/floweisshardt/atf/pull/150/commits/24ba26a2b83488bb61a08b9d2a9447f3da58d4ff